woocommerce/woocommerce-gutenberg-products-block
# Fix docs paths for actions and filters source links

Update doc hooks to output correct paths to source files in actions.md and filters.md.

Fixes #5808

### Testing

How to test the changes in this Pull Request:

1. ```npm run build:docs```
2. Open `docs/extensibility/actions.md`
3. Confirm relative Source link paths are correct.
    - Example: `[StoreApi/Utilities/CartController.php](../../src/StoreApi/Utilities/CartController.php)`
4. Open `docs/extensibility/filters.md`
3. Confirm relative Source link paths are correct.
    - Example: `[BlockTypesController.php](../../src/BlockTypesController.php)`

### Performance Impact

None.
